sal_interfaces/SALSubsystems.xml

```diff
@@ -264,8 +264,8 @@
   <SoftwareLanguage>Python for high level, Java for low level</SoftwareLanguage>
   <RuntimeLanguages>IDL</RuntimeLanguages>
   <VendorContact>Michael Rueter</VendorContact>
```

Comment:
I think we should put his company in here if he's not in business for himself.

Reply:
I think his name is actually Michael Ruder, hehehe

Reply:
Spelling his name right would be good. :-). Putting the company he works for in too I think is still a good idea.